pkg/auth/auth.go

```diff
@@ -82,6 +93,25 @@ func PersistConfluentLoginToConfig(cfg *config.Config, credentials *Credentials,
 	return addOrUpdateContext(cfg, false, credentials, ctxName, url, state, caCertPath, "", save)
 }
 
+func getSsoSub(authToken string) (string, error) {
+	token, err := jwt.ParseSigned(authToken)
+	if err != nil {
+		return "", err
+	}
+
+	var claims map[string]any
+	if err := token.UnsafeClaimsWithoutVerification(&claims); err != nil {
+		return "", err
+	}
```

Comment:
I think this code already exists somewhere else, maybe we can write a function that takes the authToken and a field like "sub" and spits out the value.

Reply:
Yeah it was in `pkg/cmd/token_validator.go`. I moved it to a new package to avoid an import cycle between cmd and auth.